doc/src/manual/calling-c-and-fortran-code.md

```diff
@@ -485,24 +486,27 @@ are not possible in the translation to Julia.
 
 Packed structs and union declarations are not supported by Julia.
 
-You can get a near approximation of a `union` if you know, a priori, the field that will have
+You can get an approximation of a `union` if you know, a priori, the field that will have
 the greatest size (potentially including padding). When translating your fields to Julia, declare
 the Julia field to be only of that type.
 
-Arrays of parameters can be expressed with `NTuple`:
+Arrays of parameters can be expressed with `NTuple`.  For example, this struct in C,
```

Comment:
As written, it seems to need a colon, but that's suboptimal because it doesn't match the second clause, which has a comma. Fixing the sentence to have parallel grammar structure resolves the  punctuation issue.